src/commands/distribute/release.ts

```diff
@@ -162,9 +164,10 @@ export default class ReleaseBinaryCommand extends AppCommand {
     }
   }
 
-  private getPrerequisites(client: AppCenterClient): Promise<[number | null, models.ExternalStoreResponse | null, Buffer, string]> {
+  private getPrerequisites(client: AppCenterClient): Promise<[number | null, models.ExternalStoreResponse | null, stream.Stream, fs.Stats, string]> {
     // load release binary file
-    const fileBuffer = this.getReleaseFileBuffer();
+    const fileStream = this.getReleaseFileStream();
+    const fileStats = this.getReleaseFileStats();
```

Comment:
(see below comment)
Here you could have:
```
const { fileStats, fileStream } = this.getReleaseFileStream();
```

Reply:
How about 1459af0 ?

Reply:
Hey that's pretty cool, but isn't there an async/await version of those calls? Rather than using the sync call, since eventually we are doing a promise.all() we should be considering promises all the way :)

Reply:
Since this just creates a stream and doesn't actually read the file, async may not be necessary. It doesn't appear to exist, probably for that reason. It's still a file system interaction, just a relatively quick one.
I'm looking at newer code than where this comment was originally placed.

Reply:
Apparently you're right, and I was confused by older code which did await.